src/UnitTests/Bug/AssignableCollectionBug.cs

```diff
@@ -38,7 +38,7 @@ protected override void Because_of()
             [Fact]
             public void Should_copy_array()
             {
-                _dest.ImageData.ShouldBeSameAs(_source.ImageData);
+                _dest.ImageData.ShouldEqual(_source.ImageData);

```

Comment:
Primitive types array are different I would say, they're just data. I wouldn't expect them to be copied.